dev/com.ibm.ws.security.wim.adapter.ldap/test/com/ibm/ws/security/wim/adapter/ldap/context/ContextManagerTest.java

```diff
@@ -714,6 +715,22 @@ public void testToString() throws Exception {
         assertTrue("Did not find Return to Primary in toString: " + toString, toString.contains("iReturnToPrimary=true"));
     }
 
+    //@Test
```

Comment:
Why is this test disabled?

Reply:
I wrote it, but realized it depends on being able to set Kerberos successfully, so I left it in to enabled later when we can run it succesfully.

Reply:
I've added a note to make sure it's enabled in my GA prep issue #15722